### PR TITLE
Changes from round of JOSS review

### PR DESCRIPTION
**Pull request by Yurlungur**
## PR Summary

This is to address comments in #427 . Thanks @guadabsb15 for the suggestions!

- Add discussion of sanity checks to getting started
- Make build/bin more consistent in documentation
- Add comment about python path
- Make sure figures display in all browsers by making png versions

## PR Checklist

- [ ] Adds a test for any bugs fixed. Adds tests for new features.
- [ ] Format your changes by using the `make format` command after configuring with `cmake`.
- [ ] Document any new features, update documentation for changes made.
- [ ] Make sure the copyright notice on any files you modified is up to date.
- [ ] After creating a pull request, note it in the CHANGELOG.md file.
- [ ] LANL employees: make sure tests pass both on the github CI and on the Darwin CI

If preparing for a new release, in addition please check the following:
- [ ] Update the version in cmake.
- [ ] Move the changes in the CHANGELOG.md file under a new header for the new release, and reset the categories.
- [ ] Ensure that any `when='@main'` dependencies are updated to the release version in the package.py

### CODE REVIEW

File: doc/sphinx/src/getting-started.rst [hunk 1]
```diff
@@ -9,16 +9,35 @@ At it's most basic, you can download and compile ``singularity-eos`` with:
 
   git clone --recursive git@github.com:lanl/singularity-eos.git
   cd singularity-eos
-  mkdir bin
-  cd bin
-  cmake -DSINGULARITY_FORCE_SUBMODULE_MODE=ON -DSINGULARITY_USE_FORTRAN=OFF ..
+  mkdir -p build && cd build
+  cmake -DSINGULARITY_FORCE_SUBMODULE_MODE=ON -DSINGULARITY_USE_FORTRAN=OFF -DSINGULARITY_BUILD_EXAMPLES=ON -DSINGULARITY_BUILD_TESTS=ON ..
   make -j
   make install # optional: install into directory defined via CMAKE_INSTALL_PREFIX
```
Comment (junghans) [inline doc/sphinx/src/getting-started.rst:15]:
```suggestion
  cmake -B builddir -S . -DSINGULARITY_FORCE_SUBMODULE_MODE=ON -DSINGULARITY_USE_FORTRAN=OFF -DSINGULARITY_BUILD_EXAMPLES=ON -DSINGULARITY_BUILD_TESTS=ON 
  cmake --build builddir --parallel
  cmake --install builddir # optional: install into directory defined via CMAKE_INSTALL_PREFIX
```
Make it more modern CMake.

Comment (Yurlungur) [reply doc/sphinx/src/getting-started.rst:15]: Do you object to using `build` instead of `builddir`?

Comment (junghans) [reply doc/sphinx/src/getting-started.rst:15]: No objection, my experience is just that user always confuse ``--build`` and `build`` in `cmake --build build`, one is an argument, one is a directory.

**Review by junghans — APPROVED** [review]

File: doc/sphinx/src/getting-started.rst [hunk 2]
```diff
@@ -9,16 +9,35 @@ At it's most basic, you can download and compile ``singularity-eos`` with:
 
   git clone --recursive git@github.com:lanl/singularity-eos.git
   cd singularity-eos
-  mkdir bin
-  cd bin
-  cmake -DSINGULARITY_FORCE_SUBMODULE_MODE=ON -DSINGULARITY_USE_FORTRAN=OFF ..
-  make -j
-  make install # optional: install into directory defined via CMAKE_INSTALL_PREFIX
+  cmake -B builddir -S . -DSINGULARITY_FORCE_SUBMODULE_MODE=ON -DSINGULARITY_USE_FORTRAN=OFF -DSINGULARITY_BUILD_EXAMPLES=ON -DSINGULARITY_BUILD_TESTS=ON
```
Comment (guadabsb15) [inline doc/sphinx/src/getting-started.rst:12]: I think that this `buildidir` should be `build`? just like the ones on line 13 and 14. I saw there was a change in name and probably that one was forgotten.

Comment (junghans) [reply doc/sphinx/src/getting-started.rst:12]: Correct, my mistake!

**Comment by Yurlungur** [comment]
Changed things back to `builddir` and did a consistency sweep through the docs. Hopefully I got them all now. I also removed the unit tests from `getting started`.

**Pull request merged**